Summator.NUnitTests/SummatorTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Summator.UnitTests
+{
+    public class SummatorTests
+    {
+        //Sum of numbers
+        [Test]
+        public static void TestSumOfTwoPositiveNumbers()
+        {
+            var numbers = new int[] { 1, 2, };
+            var actualResult = Summator.Sum(numbers);
+            var expectedResult = 3;
+            Assert.That(actualResult, Is.EqualTo(expectedResult));
+        }
+
+        [Test]
+        public static void TestSumOfTwoNegativeNumbers()
+        {
+            var numbers = new int[] { -1, -99 };
+            var actualResult = Summator.Sum(numbers);
+            var expectedResult = -100;
+            Assert.That(actualResult, Is.EqualTo(expectedResult));
+
+        }
+
+        [Test]
+        public static void TestSumOfOnePositiveNumber()
+        {
+            var numbers = new int[] { 6 };
+            var actualResult = Summator.Sum(numbers);
+            var expectedResult = 6;
+            Assert.That(actualResult, Is.EqualTo(expectedResult));
+        }
+
+        [Test]
+        public static void TestSumOfZeroNumbers()
+        {
+            var numbers = new int[] { 0 };

```

Comment:
You are actually creating an array with one element - the number 0 (zero) inside of it, and testing the sum of that array.
```suggestion
            var numbers = new int[] { 0 };
            var numbers = new int[0]; or var numbers = Array.Empty<int>()
```

Reply:
to test division by 0 there should be no elements in the array